### PR TITLE
feat(config): add Zooz Zen05 Outdoor Smart Plug

### PR DESCRIPTION
**Pull request by alvinchen1**
Added per configuration settings https://www.support.getzooz.com/kb/article/1118-zen05-outdoor-plug-advanced-settings/

### CODE REVIEW

**Comment by alvinchen1** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Let's try this again.
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Sorry for spam :)
@zwave-js-bot fix lint

File: packages/config/config/devices/0x027a/zen05.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		},
+		{
+			"#": "8",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Manual Control",
+			"defaultValue": 1
+		}
+	],
+	"compat": {
+		"preserveRootApplicationCCValueIDs": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen05.json]: Whats the reason this flag is enabled?

File: packages/config/config/devices/0x027a/zen05.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	"label": "ZEN05",
+	"description": "Outdoor Smart Plug",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xb001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "1.1",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen05.json]: Unless this is used to split a device into multiple files, min should be 
```suggestion
		"min": "0.0",
```

File: packages/config/config/devices/0x027a/zen05.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+				},
+				{
+					"label": "medium",
+					"value": 1
+				},
+				{
+					"label": "low",
+					"value": 2
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen05.json]:
```suggestion
			"$import": "templates/zooz_template.json#led_indicator_brightness"
```

File: packages/config/config/devices/0x027a/zen05.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+				{
+					"label": "Always Off",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "2",
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration "
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen05.json]:
```suggestion
			"label": "Auto Turn-Off Timer Duration"
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, I got a few remarks

File: packages/config/config/devices/0x027a/zen05.json (hunk 1 above)
Comment (alvinchen1) [reply packages/config/config/devices/0x027a/zen05.json]: Honestly, first time submitting, so tried to mimic what I saw elsewhere where possible.  Could definitely be an error on my part. Did not notice in my testing.

**Comment by alvinchen1** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x027a/zen05.json (hunk 1 above)
Comment (alvinchen1) [reply packages/config/config/devices/0x027a/zen05.json]: I think I resolved it, but did it in an inefficient stupid first-time way.  But ultimately removed the compat and re-commited.

**Review by alvinchen1 — COMMENTED** [review]
I think I did it right?  Or got there eventually.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks

**Comment by alvinchen1** [comment]
Thanks for being patient with this newbie.  Definitely a learning experience and hope to contribute more soon.

**Pull request merged**